docs/run-preinstalled-wda.md

```diff
@@ -114,3 +114,12 @@ driver = @core.start_driver
 # do something
 driver.quit
 ```
+
+> **Note**
+> As of iOS 17, the testmanagerd service name has changed to `com.apple.dt.testmanagerd.runner` from `com.apple.testmanagerd`.
+> It causes an unexpected WDA process crash with embedded XCTest frameworks to run a single WebDriverAgent package to various OS environments without `xcodebuild`.
+> As of Appium/WebDriverAgent v5.10.0, the WDA module can refer to the device's local XCTtest frameworks if the WebDriverAgent package had no these frameworks.
```

Comment:
if the WebDriverAgent package had no these frameworks -> this part is redundant

Reply:
`can refer` or `refers`?

Reply:
As of Appium/WebDriverAgent v5.10.0 -> Since appium-webdriveragent v5.10.0

Reply:
Updated.
It seems like if a package has embedded frameworks, WDA refers to them first. So it needs to remove the `XC*.framework`. Then "can refer" might be better...?